apps/omg_db/lib/omg_db/rocksdb/server.ex

```diff
@@ -115,14 +107,25 @@ defmodule OMG.DB.RocksDB.Server do
     do_get_single_value(parameter, state)
   end
 
-  def handle_call({:exit_info, utxo_pos}, _from, state) do
```

Comment:
I think we should keep the exact functions without generalizing them yet.
It works as a documentation (what kind of collections we have) and allows as to apply specific timeouts to specific functions